tests/framework/interface.go

```diff
@@ -41,6 +41,7 @@ type Client interface {
 	HashKV(rev int64) ([]*clientv3.HashKVResponse, error)
 	Health() error
 	Defragment(opts config.DefragOption) error
+	Alarm(cmd string, member *clientv3.AlarmMember) (*clientv3.AlarmResponse, error)
```

Comment:
We should follow client interface, let's have separate `AlarmList` and `AlarmDisarm` calls.

Reply:
fixed. thanks.